vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -114,6 +114,17 @@ type internal ProjectInfoManager
     member this.UpdateProjectInfo(projectId: ProjectId, site: IProjectSite, workspace: Workspace) =
         let extraProjectInfo = Some(box workspace)
         let options = ProjectSitesAndFiles.GetProjectOptionsForProjectSite(site, site.ProjectFileName(), extraProjectInfo, serviceProvider)
+
+        //lock this <| fun _ ->
```

Comment:
@vasily-kirichenko 
Could we have a comment describing why this code is commented out. It is fair to say that we all know why it is commented out right now ... but in the future we may have forgotten.

We could also consider using the preprocessor to disable it rather than using comments too.